test/evergreen/tsan_warnings_analysis.py

```diff
@@ -69,19 +69,26 @@ def get_tsan_warnings():
             if file_name.startswith("tsan_logs"):
                 file_path = os.path.join(root, file_name)  # Get the full path to the file
                 with open(file_path, "r") as file:
+
+                    start_record = False
```

Comment:
np: remove blank line?

```suggestion
                    start_record = False
```